.github/workflows/tripy-l0.yml

```diff
@@ -70,6 +70,7 @@ jobs:
         options: --gpus all -v ${{ github.workspace }}/tripy:/tripy
         run: |
           pytest --cov=tripy/ --cov-config=.coveragerc tests/ -v -m "not l1 and not manual" -n 4 --durations=15 --ignore tests/performance
+          find . -name '*.ipynb' -print0 | xargs -0 pytest --nbval-lax
```

Comment:
Any way to have this run as part of our normal pytest command? Seems like there's [another extension that can do this](https://pytest-notebook.readthedocs.io/en/latest/user_guide/tutorial_intro.html#pytest-file-collection).

Reply:
We can't really do one command, since we would need to specify another location to look for tests, `examples`. Moreover, notebook cells should be run in order so we can't use multiple workers. 

I can do this command instead
`pytest --nbval-lax examples/  -v -m "not l1 and not manual" -n 1 --durations=15`